scripts/power_control.py

```diff
@@ -52,7 +53,8 @@ def get_config():
 def power_control(VCENTER_HOST,
                   VCENTER_USER,
                   VCENTER_PASSWORD,
-                  VCENTER_ACTION):
+                  VCENTER_ACTION,
```

Comment:
Only saw now that the argument names are uppercase, they should be lower since they aren't constants.